kuma/search/jinja2/search/react.html

```diff
@@ -13,8 +13,6 @@
   {% stylesheet 'react-search' %}
 {% endblock %}
 
-{% block robots_value %}noindex, nofollow{% endblock %}
-
```

Comment:
It seems to me that this means we're changing this to `index, follow` (since it uses `react_base.html`), which seems like a mistake to me. It seems to me we should never be indexing/following any search results page.

Reply:
The base wiki site now always comes with `noindex, nofollow`, so I just removed that override from all the subpages

Reply:
I'm not so sure about that. It's a big topic but one shouldn't necessarily set `noindex,nofollow` unless you have a strong reason. A search results page (in developer.mozilla.org) is actually a perfectly fine page for Google to index. It might not be super helpful but it's not a bad page. 
It's actually not unusual to google for something and you arrive on some sort of site search results page.

Reply:
I still disagree that the search results page should be indexed/followed, but I'll happily defer to our SEO people. Here my point was simply that we've always used `noindex, nofollow` for both our read-only and wiki search results pages, and by removing this code it will inherit the robots `meta` tag from `jinja2/react_base.html` which in this PR changes to a fixed `index, follow` (the `robots_value` Jinja block has been removed).